python/paddle/fluid/dygraph/dygraph_to_static/ast_transformer.py

```diff
@@ -256,6 +344,28 @@ def _update_feed_dict(self, node):
     def get_feed_name_to_arg_id(self):
         return self.feed_name_to_arg_id
 
+    def _update_name_to_tensor_shape(self, node):
+        assert isinstance(node, gast.Assign)
+        target_node = node.targets[0]
+        value_node = node.value
+        target_id = target_node.id
```

Comment:
If  statement is  `x , y = 2, Tensor.shape[1]`,  the target_node may be `gast.Tuple` not `gast.Name`. You can add a `TODO` comment  here.

Reply:
Yes, this is a todo.  TODO comment will be added.